assert/json.go

```diff
@@ -10,6 +10,9 @@ import (
 	"reflect"
 )
 
+// convert types take an int and return a string value.
+type FnJsonVerify func(map[string]interface{}) error
```

Comment:
JSON can be a `slice` or `map`. Enforcing it as `map` structure would restrict its use cases. We should use `interface{}` type here, and let developers to perform the type assertion it as needed.

Reply:
I see your point, but in this case data come from unmarshalBody which return a map[string]interface{}